src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -849,6 +851,14 @@ public bool TryGenerateType(string possiblyQualifiedName, out IReadOnlyList<stri
                 return false;
             }
 
+            if (SpecialTypeDefNames.Contains(typeName))
```

Comment:
`SpecialTypeDefNames` happened to have exactly the needed contents. It is currently only referenced in `Generator.IsTypeDefStruct` and the contents are not even used here:

https://github.com/microsoft/CsWin32/blob/a74c60dbe9c84ccd25a24dde80fc77881fe804c7/src/Microsoft.Windows.CsWin32/Generator.cs#L4942-L4945